__tests__/Itinerary.test.js

```diff
@@ -0,0 +1,18 @@
+const { Itinerary } = require("../src/Itinerary")
+
+describe("Itinerary", () => {
+  it("can be instantiated", () => {
+    const itinerary = new Itinerary()
+    
+    expect(itinerary).toBeInstanceOf(Object)
+    expect(itinerary).toBeInstanceOf(Itinerary)
+  })
+
+  it("can have ports", () => {
+    const port1 = jest.fn()
+    const port2 = jest.fn()
```

Comment:
Why is a function being passed in as a parameter? What value is actually being passed in when an itinerary is created?